src/client/components/FavouriteList/ProductQuantity.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import './FavouriteList.styles.css';
+
+const ProductQuantity = ({ product }) => {
+  const [counter, setCounter] = useState(0);
+  const handleClick1 = () => {
+    setCounter(counter + 1);
+  };
+
+  const handleClick2 = () => {
+    setCounter(counter - 1);
```

Comment:
The counter is going to negative numbers. Try to give a condition here, so that the -counter will stop at count 1.